org.freedesktop.Sdk.Extension.openjfx.yaml

```diff
@@ -0,0 +1,43 @@
+id: org.freedesktop.Sdk.Extension.openjfx
+branch: '23.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+build-extension: true
+sdk: org.freedesktop.Sdk
+separate-locales: false
+appstream-compose: false
+build-options:
+  no-debuginfo: true
+  prefix: /usr/lib/sdk/openjfx
+
+modules:
+  - name: javafx_jmods
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://download.bell-sw.com/java/21.0.2+14/bellsoft-jdk21.0.2+14-linux-amd64-full.tar.gz
+        dest-filename: java-openjfx.tar.gz
+        sha512: 1b349703a585198af2ae56e2ec4ba1922c794be444bdf8abc6252a64a8cfe5bf26714971668ef8481f8223a3ff2a4bb14a1ba0cfe965eea1c32791e8cdb205f1
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://download.bell-sw.com/java/21.0.2+14/bellsoft-jdk21.0.2+14-linux-aarch64-full.tar.gz
+        dest-filename: java-openjfx.tar.gz
+        sha512: 043e798e02433c03779a95c9583881908a5a5e12c9c2200fd18da96a24e48103d503795b4a6c191b5cf74f4fcda2e6b04c25336799c15d59f61e6832b73bf400
+    build-commands:
+#      - tar xfv java-openjfx.tar.gz --wildcards "jdk-21.0.2-full/jmods/javafx.*.jmod" -C $FLATPAK_DEST --strip-components 1
+      - tar xfv java-openjfx.tar.gz --wildcards "jdk-21.0.2-full/jmods/javafx.*.jmod" --strip-components 1
```

Comment:
`type: archive` will auto extract it, you should remove this and adjust the commands